test/e2e/cypress/support/commands.ts

```diff
@@ -402,6 +402,51 @@ Cypress.Commands.add('recoverApi', ({ email, returnTo }) => {
     })
 })
 
+Cypress.Commands.add('verificationApiExpired', ({ email, returnTo }) => {
```

Comment:
This is not creating an expired flow, but creates one and checks if it is expired right? Do you first have to set `cy.shortVerificationLifespan()` for this to work? Could we make it part of this command then?

Reply:
Good point, I've set the short lifespan in the test before calling this, might be a better approach to include directly in the call